components/integrations/installed_incoming_webhooks/index.ts

```diff
@@ -27,20 +35,20 @@ function mapStateToProps(state) {
     const enableIncomingWebhooks = config.EnableIncomingWebhooks === 'true';
 
     return {
+        teamId,
```

Comment:
Should we just remove this prop altogether?

Reply:
Thanks @devinbinnie. I did remove this in the first push but I wasn't sure if that was correct. I think I forgot to change it back to `teamId` in my last commit. If it's confirmed that it's okay to use `team.id` instead of `teamId` in `installed_incoming_webhooks.tsx` I can remove it from here.

Reply:
Yeah I think it's fine, we'll have QA do a pass over to make sure nothing broke.

Reply:
Sounds good, addressed.